README.md

```diff
@@ -8,37 +8,55 @@ A VS Code extension for Open Liberty. The extension will detect your Liberty Mav
 ![Open Liberty Tools Extension](images/OpenLibertyTools.png)
 
 ## Quick Start
+
 - Install the extension
 - Open Liberty supported projects will appear in the Liberty Dev Dashboard on the side bar
 - Right-click a project in the Liberty Dev Dashboard to view the available commands
 
 ## Features
+
 - View supported `liberty-maven-plugin`(version `3.1` or higher) or `liberty-gradle-plugin`(version `3.0` or higher) projects in the workspace
 - Start/Stop dev mode
 - Start dev mode with custom parameters
 - Run tests
 - View unit and integration test reports
 
+## Commands
+
+| Command                      | Description                                                                                                                                                                                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Run tests                    | Runs the unit tests and integration tests that are configured for your project. This command requires dev mode to be already started.                                                                                                                                                                                        |
+| Start                        | Starts dev mode.                                                                                                                                                                                                                                                                                                             |
+| Start…​                      | Starts dev mode with custom parameters. For example, to run tests automatically after every change, include the -DhotTests parameter. Additional supported parameters can be found in the documentation for the [dev goal of the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev). |
```

Comment:
Parameter could be either Maven or Gradle format depending on if it's a Maven or Gradle project.  If providing links, also link to Gradle libertyDev task.